bundles/fluent-en_GB/atrium-fluent-en_GB-jvm/src/test/kotlin/custom/SmokeSpec.kt

```diff
@@ -8,19 +8,22 @@ import ch.tutteli.atrium.domain.builders.ExpectImpl
 import ch.tutteli.atrium.reporting.RawString
 import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
 import ch.tutteli.atrium.translations.DescriptionBasic
-import org.jetbrains.spek.api.Spek
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
 
 object SmokeSpec : Spek({
-    test("see if `toBe` can be used") {
-        assertThat(1).toBe(1)
-    }
-
-    test("see if own assertion function without i18n can be used") {
-        assertThat(2).isEven()
-    }
-
-    test("see if own assertion function with i18n can be used") {
-        assertThat(4).isMultipleOf(2)
+    describe("Smoke tests") {
```

Comment:
I think this description does not add much and only clutters the code. I would keep `test(...){ ... }` instead of `describe(...) {  it(..){ .. } }` but up to you.

Reply:
Ok, I will change it back, I thought that in spek2 test block was replaced with it block but its worked anyway

Reply:
Committed